apps/site/src/app/(main)/resources/page.tsx

```diff
@@ -0,0 +1,59 @@
+import Button from "@/lib/components/Button/Button";
+import { Metadata } from "next";
+import { redirect } from "next/navigation";
+import { getResources } from "./getResources";
+
+export const revalidate = 60;
+
+export const metadata: Metadata = {
+	title: "Resources | IrvineHacks 2024",
+};
+
+export default async function Schedule() {
+	if (process.env.MAINTENANCE_MODE_SCHEDULE) {
+		redirect("/");
+	}
+
```

Comment:
Issue: Could you rename the function to `Resources` and then change the variable name to `MAINTENANCE_MODE_RESOURCES`?